File: server/proj/proj/src/main/java/com/capstone/proj/vote/Vote.java
package com.capstone.proj.vote;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.util.Objects;

@Getter
@Setter
@EqualsAndHashCode
@ToString
public class Vote {
    private Integer id;
    private String issue;
    private String details;
    private Integer mpId;
    private Boolean votedFor;

    public Vote(Integer id, String issue, String details, Integer mpId, Boolean votedFor) {
        this.issue = issue;
        this.details = details;
        this.mpId = mpId;
        this.votedFor = votedFor;
    }

}

File: client/my-app/src/components/mp/Vote.js
const Vote = ({ vote }) => {

    return (
        <>
            <tr>
                <td>
                    <h2>{vote.title}</h2>
                </td>

                <td>
                    <h2>{vote.vote===true? "✅": "❌"}</h2>
                </td>
            </tr>
        </>
    )
}

export default Vote;